lab_5_scrapper/scrapper_config.json

```diff
@@ -1,9 +1,37 @@
 {
-    "seed_urls": [],
-    "headers": {},
-    "total_articles_to_find_and_parse": 0,
-    "encoding": "",
-    "timeout": 0,
+    "seed_urls": ["https://dzer.ru/korotko-o-glavnom/",
+    "https://dzer.ru/korotko-o-glavnom/?PAGEN_1=2",
+    "https://dzer.ru/korotko-o-glavnom/?PAGEN_1=3",
+    "https://dzer.ru/korotko-o-glavnom/?PAGEN_1=4",
+    "https://dzer.ru/korotko-o-glavnom/?PAGEN_1=5",
+    "https://dzer.ru/korotko-o-glavnom/?PAGEN_1=6",
+    "https://dzer.ru/korotko-o-glavnom/?PAGEN_1=7",
+    "https://dzer.ru/korotko-o-glavnom/?PAGEN_1=8",
+    "https://dzer.ru/korotko-o-glavnom/?PAGEN_1=9",
+    "https://dzer.ru/korotko-o-glavnom/?PAGEN_1=10",
+    "https://dzer.ru/korotko-o-glavnom/?PAGEN_1=11",
+    "https://dzer.ru/korotko-o-glavnom/?PAGEN_1=12",
+    "https://dzer.ru/korotko-o-glavnom/?PAGEN_1=13",
+    "https://dzer.ru/korotko-o-glavnom/?PAGEN_1=14",
+    "https://dzer.ru/korotko-o-glavnom/?PAGEN_1=15",
+    "https://dzer.ru/korotko-o-glavnom/?PAGEN_1=16",
+    "https://dzer.ru/korotko-o-glavnom/?PAGEN_1=17"],
+    "headers": {
+        "accept": "text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.7",
+        "accept-encoding": "gzip, deflate, br",
+        "accept-language": "ru-RU,ru;q=0.9,en-US;q=0.8,en;q=0.7",
+        "cache-control": "max-age=0",
+        "cookie": "_ym_uid=167964123180194909; _ym_d=1679641231; BX_USER_ID=f440a206a5d601bc2e412d19b6a5277c; PHPSESSID=ITWd9TgKEH3voJfEHVypYRRpyKQq6458; _ym_isad=2; _ym_visorc=w",
+        "sec-fetch-dest": "document",
+        "sec-fetch-mode": "navigate",
+        "sec-fetch-site": "none",
+        "sec-fetch-user": "?1",
+        "upgrade-insecure-requests": "1",
+        "user-agent": "Mozilla/5.0 (iPhone; CPU iPhone OS 13_2_3 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.0.3 Mobile/15E148 Safari/604.1"
+    },
+        "total_articles_to_find_and_parse": 10,
```

Comment:
better layout. use PyCharm or external services for json layouting

Reply:
fixed